### PR TITLE
Implemeneted the cloud resource orchrestration long_run resource

### PR DESCRIPTION
**Pull request by athiruma**
1. Manage the AWS ec2 long-run management.
2. Tag the instances which have the JiraId and move the Jira ticket to in progress.
3. Monitor the instances and upload the state, and price of the instance to the elastic search.
4. Extend the long run days, if the user opens the extend long run days.
5. Close the in-progress Jira ticket if no instance is present.
6. Added the cloud_data_ware_house for landing page data

### CODE REVIEW

File: cloud_governance/main/main.py [hunk 1]
```diff
@@ -214,6 +221,10 @@ def main():
         if is_azure_policy_runner:
             azure_cost_policy_runner = AzurePolicyRunner()
 
+    is_cloud_management = False
+    if environment_variables_dict.get('MANAGEMENT'):
```
Comment (athiruma) [inline cloud_governance/main/main.py:225]: Add Documentation for short run/long run

File: Dockerfile [hunk 2]
```diff
@@ -18,6 +18,7 @@ RUN apt-get update \
 RUN python -m pip --no-cache-dir install --upgrade pip && pip --no-cache-dir install cloud-governance --upgrade
 RUN pip3 --no-cache-dir install --upgrade awscli
 ADD cloud_governance/policy /usr/local/cloud_governance/policy/
+ADD cloud_governance/cloud_resource_orchestration /usr/local/cloud_governance/cloud_resource_orchestration
```
Comment (athiruma) [inline Dockerfile:21]: Remove this line and verify it is working

File: cloud_governance/cloud_resource_orchestration/aws/long_run/ec2_long_run.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            if self.__es_upload.elastic_search_operations.verify_elastic_index_doc_id(index=self.__es_index, doc_id=jira_id):
+                es_data = self.__es_upload.elastic_search_operations.get_es_data_by_id(id=jira_id, index=self.__es_index)
+                source = es_data.get('_source')
+                instance_ids = source.get('instance_ids', [])
+                update_es_data = {'cost_estimation': cost_estimation,
+                                  'long_run_days': int(instances[0].get('long_run_days')),
+                                  'total_instances': len(instances)}
+                running_days = 0
+                total_price = 0
+                for instance in instances:
```
Comment (athiruma) [inline cloud_governance/cloud_resource_orchestration/aws/long_run/ec2_long_run.py:58]: Data preparation in dedicated method

File: cloud_governance/cloud_resource_orchestration/aws/long_run/tag_long_run.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                self.jira_operations.move_issue_state(jira_id=task_id, state='closed')
+
+    @logger_time_stamp
+    def tag_jira_id_attach_instance(self, jira_id: str, instance_id: str, volume_ids: list):
+        """
+        This method tag the long run instance with tags
+        """
+        jira_description = self.jira_operations.get_issue_description(jira_id=jira_id, state='31')
+        if jira_description:
+            if 'CLGOVN' not in jira_id:
```
Comment (athiruma) [inline cloud_governance/cloud_resource_orchestration/aws/long_run/tag_long_run.py:68]: Use env varibale

File: cloud_governance/cloud_resource_orchestration/aws/long_run/tag_long_run.py [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                self.__es_upload.elastic_search_operations.update_elasticsearch_index(metadata=data, id=jira_id, index=self.__es_upload.es_index)
+            for task_id in sub_tasks:
+                self.jira_operations.move_issue_state(jira_id=task_id, state='closed')
+
+    @logger_time_stamp
+    def tag_jira_id_attach_instance(self, jira_id: str, instance_id: str, volume_ids: list):
+        """
+        This method tag the long run instance with tags
+        """
+        jira_description = self.jira_operations.get_issue_description(jira_id=jira_id, state='31')
```
Comment (athiruma) [inline cloud_governance/cloud_resource_orchestration/aws/long_run/tag_long_run.py:66]: Add import here of jira status

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**